src/components/PaperLayout/paper.module.css

```
@@ -0,0 +1,301 @@
+/* ============================================================
+   PaperLayout — shared warm theme
+   All color values reference CSS variables from tokens.css.
+   ============================================================ */
+
+/* ── Page wrapper ──────────────────────────────────────── */
+.page {
+  background: var(--paper-bg);
+  color: var(--paper-text);
+  font-family: 'Geist', -apple-system, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)